docs/commands/help.md

```diff
@@ -1,5 +1,7 @@
-## Command help ##
+## Command gef help ##
 
-Displays the help menu for the commands loaded.
+Displays the help menu for the loaded GEF commands.
 
-![help](https://i.imgur.com/cvMoI3P.png)
```

Comment:
Why remove the screenshot?

Reply:
I can also replace it with a current one if you want. I just thought that a screenshot with a list of GEF commands will be hard to keep up-to-date at all times.

Reply:
Naw, it's fine, just wondering your reasoning.